CHANGES.txt

```diff
@@ -44,6 +44,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Ninja: Fix issue where Configure files weren't being properly processed when build run
       via ninja.
     - Added ninja mingw support and improved ninja CommandGeneratorAction support.
+    - Added contribution from MongoDB ninja alias generated source tagging:
```

Comment:
Need something more verbose here, if people have to go look at another repo's commits.. they'll never do it.

Reply:
done 1857567